src/Drupal/Commands/core/FieldDeleteCommands.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Drush\Drupal\Commands\core;
+
+use Drupal\Core\Entity\EntityTypeBundleInfo;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\FieldConfigInterface;
+use Drush\Commands\DrushCommands;
+use Symfony\Component\Console\Input\InputOption;
+
+class FieldDeleteCommands extends DrushCommands
+{
+    use AskBundleTrait;
+    use ValidateEntityTypeTrait;
+
+    /** @var EntityTypeManagerInterface */
+    protected $entityTypeManager;
+    /** @var EntityTypeBundleInfo */
+    protected $entityTypeBundleInfo;
+
+    public function __construct(
+        EntityTypeManagerInterface $entityTypeManager,
+        EntityTypeBundleInfo $entityTypeBundleInfo
+    ) {
+        $this->entityTypeManager = $entityTypeManager;
+        $this->entityTypeBundleInfo = $entityTypeBundleInfo;
+    }
+
+    /**
+     * Delete a field
+     *
+     * @command field:delete
+     * @aliases field-delete,fd
+     *
+     * @param string $entityType
+     *      The machine name of the entity type
+     * @param string $bundle
+     *      The machine name of the bundle
+     *
+     * @option field-name
+     *      The machine name of the field
+     *
+     * @option show-machine-names
+     *      Show machine names instead of labels in option lists.
+     *
+     * @usage drush field:delete
```

Comment:
Looks like we dont yet support omitting entityType. Same for field:info. I think we should. Any chance you can add this @DieterHolvoet? I've just pushed some tests.

Reply:
Good idea, I just added this to `field:delete`, `field:create` and `field:info`.